### PR TITLE
feat: tier property now free text field to allow custom dbs

### PR DESCRIPTION
**Pull request by jimbo459**
[#181805808](https://www.pivotaltracker.com/story/show/181805808)

### Checklist:

* [ ] Have you added Draft Release Notes in `docs/draft-release-notes.md`?
* [x] Have you followed the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,7 +19,7 @@ PAK_CACHE=/tmp/.pak-cache
...
+export GSB_SERVICE_CSB_GOOGLE_POSTGRES_PLANS = [{"name":"small","id":"85b27a04-8695-11ea-818a-274131861b81","description":"PostgreSQL with default version, shared CPU, minimum 0.6GB ram, 10GB storage","display_name":"small","tier":"db-f1-micro","storage_gb":10},{"name":"medium","id":"b41ee300-8695-11ea-87df-cfcb8aecf3bc","description":"PostgreSQL with default version, shared CPU, minimum 1.7GB ram, 20GB storage","display_name":"medium","tier":"db-g1-small","storage_gb":20},{"name":"large","id":"2a57527e-b025-11ea-b643-bf3bcf6d055a","description":"PostgreSQL with default version, minimum 8 cores, minimum 8GB ram, 50GB storage","display_name":"large","tier":"db-custom-8-8192","storage_gb":50}]
```
Comment (pivotal-marcela-campo) [inline Makefile:22]: It looks like this is reverting the change to GUIDs that we just did to release the GA

File: google-postgresql.yml [hunk 2]
```diff
@@ -30,24 +30,6 @@ provision:
     required: true
     type: string
     details: Google sql tier. This determines CPU and RAM resource.
```
Comment (pivotal-marcela-campo) [inline google-postgresql.yml:32]: Is there an example of how the field should look like that we could add? Or a link to gcp docs?

**Review by pivotal-marcela-campo — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (jimbo459) [reply Makefile]: Thanks Marcela - good catch

File: google-postgresql.yml (hunk 2 above)
Comment (jimbo459) [reply google-postgresql.yml]: Yes good shout - I have added a link to the gcp docs.

**Review by pivotal-marcela-campo — APPROVED** [review]

**Pull request merged**